operationsgateway_api/src/main.py

```diff
@@ -71,7 +72,13 @@ async def get_experiments_on_startup():
         else:
             log.info("Scheduler background task has not been enabled")
 
+    @assign_event_to_single_worker()
+    async def set_bucket_expiry() -> None:
```

Comment:
Does this need to async? `put_lifecycle` uses the sync boto lib.

Reply:
Yeah I think you're right; I can't recall any reason it would need to so probably a copy paste from `get_experiments_on_startup` above.

Having said that, if/when #199 is merged then it might end up needing to change to async anyway. Depending on what order things get merged I'll sort it out in the conflict resolution one way or another.